src/tvdb/generated_models.py

```diff
@@ -0,0 +1,750 @@
+# generated by datamodel-codegen:
+#   filename:  swagger.yml
+#   timestamp: 2024-07-20T12:48:32+00:00
```

Comment:
Would you mind maybe adding some instructions on how to generate these into the readme? It may be worth it to add a simple bash script to re-generate these if needed (in case the API is updated). Also, maybe the `swagger.yml` can be gitignored if it's just used to generate this? It can always be re-pulled (perhaps from that script)

Reply:
Will do